components/Button/ButtonPrimary.tsx

```diff
@@ -0,0 +1,20 @@
+import Button from "./buttonPrimary.style";
+import Link from "next/link";
+import React from 'react'
+
+interface Buttons {
+  text: string
+  href?: string
+}
+
+const ButtonPrimary = ({ text, href }:Buttons) => {
```

Comment:
The component will be Button, not ButtonPrimary. Then, if we want to have different variants then we will pass a prop with the variant we want for example type="primary" or type="secondary" etc.